provider/win_wora.py

```diff
@@ -0,0 +1,33 @@
+"""
+windows workaround functions.
+
+"""
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def modify_driver(params, session):
+    """
+    Add a workaround for win2016 guest to solve the issue that
+    "HID Button over Interrupt Driver" does occupy hot-added cpu's driver,
+    cause the system cannot detected new added cpu, need modify the driver.
+    issue details please refer to:
+    https://support.huawei.com/enterprise/zh/doc/EDOC1100034211/5ba99a60.
+    """
+    devcon_path = utils_misc.set_winutils_letter(session,
+                                                 params["devcon_path"])
+    dev_hwid = params["dev_hwid"]
+    chk_cmd = '%s find %s' % (devcon_path, dev_hwid)
+    chk_pat = r'ACPI\\ACPI0010.*\: Generic Bus'
+    while not re.search(chk_pat, session.cmd(chk_cmd)):
```

Comment:
Is there any reason to use `while`?

Reply:
Want to run twice chk_cmd command here, first time, check if the correct driver is installed, if not, run install command; second time, double check if the correct driver is installed.

Reply:
Okay, thanks. Only one note: if the driver installation failed, this `while` loop will never terminate.

Reply:
Prefer to raise error if installation failed instead of keeping running endless loop.

Reply:
Updated, thanks~